backend/plugins/posclient_api/src/modules/posclient/graphql/resolvers/mutations/configs.ts

```
@@ -0,0 +1,234 @@
+      timeout: 300000,
+      method: 'POST',
+    });
+
+    if (!response.ok) {
+      throw new Error(response.statusText);
+    }
+    const responseData = await response.json();
+
+    switch (type) {
+      case 'config':
+        const { pos = {}, adminUsers = [], cashiers = [] } = responseData;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)